src/08-tasks/tasks.apib

```diff
@@ -31,6 +31,10 @@ Get a list of tasks.
                 + completed: false (boolean)
                 + completed_at: `2016-02-04T16:44:33+00:00` (string)
                 + due_at: `2016-02-04T16:44:33+00:00` (string)
+                + estimated_duration: 3600 (number) - In seconds
```

Comment:
B: We should use an object for duration fields from now on. See https://github.com/teamleadercrm/api/pull/299

Reply:
I know, but i though that change would be made after the code was merged. Since the current code of tasks.list and tasks.info doesn't use the object yet.

Reply:
The code doesn't use the object yet indeed, but the documentation is using the new format everywhere except here, unless I'm mistaken. At the moment the code does not conform with the documentation, but I'm taking care of that.